BrainPortal/app/views/nh_projects/show_license.html.erb

```diff
@@ -0,0 +1,109 @@
+
+<%-
+  #
+  # CBRAIN Project
+  #
+  # Copyright (C) 2008-2012
+  # The Royal Institution for the Advancement of Learning
+  # McGill University
+  #
+  # This program is free software: you can redistribute it and/or modify
+  # it under the terms of the GNU General Public License as published by
+  # the Free Software Foundation, either version 3 of the License, or
+  # (at your option) any later version.
+  #
+  # This program is distributed in the hope that it will be useful,
+  # but WITHOUT ANY WARRANTY; without even the implied warranty of
+  # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  # GNU General Public License for more details.
+  #
+  # You should have received a copy of the GNU General Public License
+  # along with this program.  If not, see <http://www.gnu.org/licenses/>.
+  #
+-%>
+
+<% is_signed = current_user.all_custom_licenses_signed.include?(@license) %>
+
+  <% if ! is_signed %>
+    <%= form_tag :action => :sign_license, :license => @license %>
+    <%= hidden_field_tag :license, @license %>
+  <% end %>
+  <p>
+
+<fieldset>
+  <legend>Carefully read the license agreement below.</legend>
+
+  <% if @userfile.archived? %>
+    <%= "This document has been archived." %><br>
+    Content viewers are disabled until the file is unarchived.
+
+      (This file cannot be viewed as it is stored on Data Provider
+      <%= link_to_data_provider_if_accessible(@userfile.data_provider) %>
+      which is configured to not allow synchronization at all)
+
+    <% elsif @sync_status == "Corrupted" %>
```

Comment:
Spacing issue.

Reply:
could you hint what should be correct spacing

Reply:
@natacha-beck

Reply:
The start of each line is aligned well. For example the start of the `if` have 2 spacing indentations but the `elsif` after that have 4 space @MontrealSergiy